homeassistant/components/tradfri/switch.py

```diff
@@ -72,12 +72,10 @@ def is_on(self) -> bool:
 
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Instruct the switch to turn off."""
-        if not self._device_control:
-            return
```

Comment:
Why do we invert the guard clauses? Those aren't related to this ruff rule right?

Reply:
No, I just thought that we can get rid of the return statements.